app/assets/stylesheets/header.scss

```diff
@@ -0,0 +1,17 @@
+.header {
+  height: 30px;
+  background-color: rgba(0,0,0,0.75);
+  color: #fff;
+  padding: 5px;
+
+  .resources {
+    float: left;
+  }
+
+  .user-actions {
+    float: right;
+    a {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3